File: src/services/tree-sitter/__tests__/fixtures/csharp/sample.cs
using System;

namespace SampleNamespace
{
    public class CSharpClass
    {
        public string Name { get; set; }

        public CSharpClass(string name)
        {
            Name = name;
        }

        public string GetName()
        {
            return Name;
        }

        public int Calculate(int a, int b)
        {
            return a + b;
        }
    }

    public static class Program
    {
        public static void Main(string[] args)
        {
            var obj = new CSharpClass("example");
            Console.WriteLine(obj.GetName());
            Console.WriteLine(obj.Calculate(5, 10));
        }

        public static int TopLevelFunc(int x)
        {
            var obj = new CSharpClass("top-level");
            return obj.Calculate(x, 10);
        }
    }
}

File: src/services/tree-sitter/__tests__/fixtures/php/sample.php
<?php

namespace SampleNamespace;

/**
 * PhpClass is a sample class.
 */
class PhpClass
{
    private string $name;

    public function __construct(string $name)
    {
        $this->name = $name;
    }

    /**
     * getName returns the name.
     */
    public function getName(): string
    {
        return $this->name;
    }

    public function calculate(int $a, int $b): int
    {
        return $a + $b;
    }
}

/**
 * top_level_func is a top-level function.
 */
function top_level_func(int $x): int
{
    $obj = new PhpClass("example");
    echo $obj->getName();
    return $obj->calculate($x, 10);
}

top_level_func(5);

class AnotherClass {
    public static function staticMethod() {
        return "static";
    }
}
